docs/synapse/userguides/ug018_storm_ref_misc.rst

```diff
@@ -13,7 +13,54 @@ Todo
 
 limit()
 -------
-Todo
+Provides a hard limit on the number of nodes which are emitted by the ``limit()`` operator.
+This also back-propagates a ``limit=`` argument to the previous query operator.
+
+
+Optional parameters:
```

Comment:
I think "Optional parameters" section is unnecessary; there aren't any parameters (other than the value specified for < limit > )

Reply:
Thats what I get for copy pasta

Reply:
removed